pkgs/development/python-modules/isoweek/default.nix

```
@@ -0,0 +1,21 @@
+{ stdenv, fetchPypi, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "isoweek";
+  version = "1.3.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1s7zsf0pab0l9gn6456qadnz5i5h90hafcjwnhx5mq23qjxggwvk";
+  };
+
+  doCheck = false;
+
+  meta = with stdenv.lib; {
```

```suggestion
  meta = with lib; {
```
If you're only using `stdenv.lib` you might as well just use `lib` directly here.